.github/workflows/msvc.yml

```
@@ -0,0 +1,18 @@
+name: Visual Studio build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: microsoft/setup-msbuild@v1.1
+    - name: MSBuild x86
+      run: msbuild msvc\libusb.sln /p:Configuration=Release /p:Platform=Win32
+    - name: MSBuild x64
+      run: msbuild msvc\libusb.sln /p:Configuration=Release /p:Platform=x64
+    - name: MSBuild ARM
+      run: msbuild msvc\libusb.sln /p:Configuration=Release /p:Platform=ARM
+    - name: MSBuild ARM64
+      run: msbuild msvc\libusb.sln /p:Configuration=Release /p:Platform=ARM64
```

this one is a perfect place to use a Matrix:
```suggestion
    strategy:
      matrix:
        platform: [Win32, x64, ARM, ARM64]
    - name: MSBuild ${{ matrix.platform }}
      run: msbuild msvc\libusb.sln /p:Configuration=Release /p:Platform=${{ matrix.platform }}
```
maybe, even, multiplied by the configuration (later?)